### ISSUE TITLE
8$: error on multiple 'export default'-s

### ISSUE THREAD

**Issue by strager** [quick-lint/quick-lint-js#1039]
```javascript
export default class A {}
export default class B {}
```
The above code is invalid. quick-lint-js should report something like the following:
> line 2: error: cannot export default more than once
> line 1: note: export default previously appeared here

**Comment by k1ng440** [comment]
I want to give this a try. I can't promise a good clean code though and I will not accept payment.
